src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -55,6 +71,20 @@ public String getDescription(Field field) {
                     return Localization.lang("Edition of a printed publication.");
                 case EDITOR:
                     return Localization.lang("Editor(s) of the work or the main publication, depending on the type of the entry.");
+                case EDITORA:
+                    return Localization.lang("A secondary editor performing a different editorial role, such as compiling, redacting, etc.");
+                case EDITORB:
+                    return Localization.lang("Another secondary editor performing a different role.");
+                case EDITORC:
+                    return Localization.lang("Another secondary editor performing a different role.");
+                case EDITORTYPE:
+                    return Localization.lang("The type of editorial role performed by the editor. Roles supported by default are \"editor\", \"compiler\", \"founder\", \"continuator\", \"redactor\", \"reviser\", \"collaborator\", \"organizer\". The role \"editor\" is the default. In this case, the field is omissible.");
+                case EDITORATYPE:
+                    return Localization.lang("Similar to \"Editortype\" but referring to the editora field.");
```

Comment:
Just copy the text above (otherwise the user has to look at the tooltip for editortype...)